03-aad.md

```diff
@@ -66,16 +74,26 @@ Following the steps below you will result in an Azure AD configuration that will
    ```bash
    az ad group member add -g $AADOBJECTID_GROUP_CLUSTERADMIN --member-id $AADOBJECTID_USER_CLUSTERADMIN
    ```
+    This object ID will be used later while creating the cluster. This way, once the cluster gets deployed the new group will get the proper Cluster Role bindings in Kubernetes. <br/>
+   Add executer to the cluster admin security group.
+
+   ```bash
+   export SIGNEDIN_USER_ID=$(az ad signed-in-user show --query objectId -o tsv)
+
+   az ad group member add -g $AADOBJECTID_GROUP_CLUSTERADMIN --member-id $SIGNEDIN_USER_ID
+   ``` 
+
 
-   This object ID will be used later while creating the cluster. This way, once the cluster gets deployed the new group will get the proper Cluster Role bindings in Kubernetes.
 
-1. Set up groups to map into other Kubernetes Roles. _Optional, fork required._
+1.    >:exclamation:OCW NOTE: This step is optional.
+   
+      Set up groups to map into other Kubernetes Roles. _Optional, fork required._
 
-   > :book: The team knows there will be more than just cluster admins that need group-managed access to the cluster. Out of the box, Kubernetes has other roles like _admin_, _edit_, and _view_ which can also be mapped to Azure AD Groups for use both at namespace and at the cluster level.
+      > :book: The team knows there will be more than just cluster admins that need group-managed access to the cluster. Out of the box, Kubernetes has other roles like _admin_, _edit_, and _view_ which can also be mapped to Azure AD Groups for use both at namespace and at the cluster level.
 
-   In the [`cluster-rbac.yaml` file](./cluster-manifests/cluster-rbac.yaml) and the various namespaced [`rbac.yaml files`](./cluster-manifests/cluster-baseline-settings/rbac.yaml), you can uncomment what you wish and replace the `<replace-with-an-aad-group-object-id...>` placeholders with corresponding new or existing AD groups that map to their purpose for this cluster or namespace. You do not need to perform this action for this walk through; they are only here for your reference.
+      In the [`cluster-rbac.yaml` file](./cluster-manifests/cluster-rbac.yaml) and the various namespaced [`rbac.yaml files`](./cluster-manifests/cluster-baseline-settings/rbac.yaml), you can uncomment what you wish and replace the `<replace-with-an-aad-group-object-id...>` placeholders with corresponding new or existing AD groups that map to their purpose for this cluster or namespace. You do not need to perform this action for this walk through; they are only here for your reference.
```

Comment:
Are any changes needed in the yaml files to support the changes that we are making in these directions?